.github/workflows/test.yml

```diff
@@ -17,6 +17,6 @@ on:
 jobs:
   test:
     if: github.repository == 'remix-run/remix'
-    uses: remix-run/remix/.github/workflows/reusable-test.yml@main
+    uses: jenseng/remix/.github/workflows/reusable-test.yml@pin-npm-install-workflow-step
```

Comment:
converted to a draft due to this testing chicken/egg problem ... temporarily setting this so the change actually gets tested :)